### ISSUE TITLE
Syntax highlighting

### ISSUE THREAD

**Issue by pieter-edelman-nictiz** [health-validator/Hammer#63]
It would be cool if the text window would offer syntax highlighting for XML/JSON.

**Comment by vadi2** [comment]
I found a good syntax highlighting library for C# but the issue with doing this is that we need access to the raw, non-HTML text via textDocument() - which we can't from C# because it's not an invokable method.

I think it would either involve [unmanaged hosting](https://qmlnet.github.io/setup/unmanaged-hosting/) or extensing QmlNet's bindings by hand. I'm not sure yet which approach is the best.

**Comment by pieter-edelman-nictiz** [comment]
Maybe it's best to postpone this to the '2.x version' then. There's no immediate need and maybe other cases will present themselves that will guide the choice for the one or the other option.

**Comment by vadi2** [comment]
Blocked by https://github.com/qmlnet/qmlnet/issues/147.

**Comment by vadi2** [comment]
@pieter-edelman-nictiz Do you want to try tackling this? I feel like it's the next big request in Hammer and I couldn't make it work, but maybe you have fresh insight?

I want to work on packaging for the next few days for macOS and Linux so people don't have to pre-install .NET Core first (if possible)

**Comment by pieter-edelman-nictiz** [comment]
Sure, but it won't be anytime soon then ...
